server/store/middleware/infohash/README.md

```diff
@@ -0,0 +1,69 @@
+## Infohash Blacklisting/Whitelisting Middlewares
+
+This package provides the announce middleware `infohash_blacklist` and `infohash_whitelist` for blacklisting or whitelisting infohashes for announces and the configurable scrape middleware `infohash_blacklist` and `infohash_whitelist` for blacklisting or whitelisting infohashes for scrape requests.
+
+### `infohash_blacklist`
+
+#### For announces
+
+The `infohash_blacklist` middleware uses all infohashes stored in the `StringStore` with a `PrefixInfohash` prefix to blacklist, i.e. block announces.
```

Comment:
This is confusing. Is the prefix literally `PrefixInfohash` or is PrefixInfohash a variable?